src/node/cli.ts

```diff
@@ -84,7 +84,10 @@ type Options<T> = {
 
 const options: Options<Required<Args>> = {
   auth: { type: AuthType, description: "The type of authentication to use." },
-  password: { type: "string", description: "The password for password authentication." },
+  password: {
+    type: "string",
+    description: "The password for password authentication (can only be passed in via $PASSWORD or the config file).",
```

Comment:
Hopefully I didn't miss it but I didn't see anything preventing this flag from being used.

Reply:
Yea lol my bad.